### PR TITLE
Fixed wrong or missing PHPDoc

### PR DESCRIPTION
**Pull request by core23**
### Subject
Some PHPDoc annotations are wrong or missing.

### CODE REVIEW

File: Bundle/OdmMetadata.php [hunk 1]
```diff
@@ -15,12 +15,36 @@
...
+     */
     protected $extendedDocumentDirectory;
+
+    /**
+     * @var string
+     */
     protected $extendedSerializerDirectory;
 
+    /**
+     * OdmMetadata constructor.
```
Comment (Soullivaneuh) [inline Bundle/OdmMetadata.php]: Useless comment.

File: Bundle/OrmMetadata.php [hunk 2]
```diff
@@ -15,12 +15,36 @@
...
+     */
     protected $extendedEntityDirectory;
+
+    /**
+     * @var string
+     */
     protected $extendedSerializerDirectory;
 
+    /**
+     * OrmMetadata constructor.
```
Comment (Soullivaneuh) [inline Bundle/OrmMetadata.php]: Useless comment.

File: Bundle/PhpcrMetadata.php [hunk 3]
```diff
@@ -15,12 +15,36 @@
...
+     */
     protected $extendedDocumentDirectory;
+
+    /**
+     * @var string
+     */
     protected $extendedSerializerDirectory;
 
+    /**
+     * PhpcrMetadata constructor.
```
Comment (Soullivaneuh) [inline Bundle/PhpcrMetadata.php]: Useless comment.

File: Command/DumpMappingCommand.php [hunk 4]
```diff
@@ -41,6 +42,7 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        /** @var ClassMetadataFactory $factory */
```
Comment (Soullivaneuh) [inline Command/DumpMappingCommand.php]: I'm not sure this is necessary. This is already given by service resolution.

File: Generator/OdmGenerator.php [hunk 5]
```diff
@@ -16,9 +16,19 @@
...
+     */
     protected $documentTemplate;
+
+    /**
+     * @var string
+     */
     protected $documentRepositoryTemplate;
 
+    /**
+     * OdmGenerator constructor.
```
Comment (Soullivaneuh) [inline Generator/OdmGenerator.php]: Useless comment.

File: Generator/OrmGenerator.php [hunk 6]
```diff
@@ -16,18 +16,27 @@
...
+     */
     protected $entityTemplate;
+
+    /**
+     * @var string
+     */
     protected $entityRepositoryTemplate;
 
+    /**
+     * OrmGenerator constructor.
```
Comment (Soullivaneuh) [inline Generator/OrmGenerator.php]: Useless comment.

File: Generator/PHPCRGenerator.php [hunk 7]
```diff
@@ -16,18 +16,27 @@
...
+     */
     protected $DocumentTemplate;
+
+    /**
+     * @var string
+     */
     protected $DocumentRepositoryTemplate;
 
+    /**
+     * PHPCRGenerator constructor.
```
Comment (Soullivaneuh) [inline Generator/PHPCRGenerator.php]: Useless comment.

File: Generator/SerializerGenerator.php [hunk 8]
```diff
@@ -16,18 +16,27 @@
...
+     */
     protected $entitySerializerTemplate;
+
+    /**
+     * @var string
+     */
     protected $documentSerializerTemplate;
 
+    /**
+     * SerializerGenerator constructor.
```
Comment (Soullivaneuh) [inline Generator/SerializerGenerator.php]: Useless comment.

File: Mapper/DoctrineCollector.php [hunk 9]
```diff
@@ -13,20 +13,44 @@
...
+     */
     protected $inheritanceTypes;
 
+    /**
+     * @var DoctrineCollector
+     */
     private static $instance;
 
+    /**
+     * DoctrineCollector constructor.
```
Comment (Soullivaneuh) [inline Mapper/DoctrineCollector.php]: Useless comment.

**Comment by core23** [comment]
Please review @greg0ire @Soullivaneuh @OskarStark

**Comment by OskarStark** [comment]
LGTM 👍

**Comment by OskarStark** [comment]
@Soullivaneuh @greg0ire please give a final review, thank you!

**Comment by greg0ire** [comment]
Thanks @core23 !

**Pull request merged**